src/components/Table/TableWrapper.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+import { prop } from "styled-tools";
+import as from "../../enhancers/as";
+import Base from "../Base";
+
+const TableWrapper = styled(Base)`
+	width: 100%;
+	overflow-x: auto;
+	-webkit-overflow-scrolling: touch;
+	${prop("theme.TableWrapper")};
+`;
+
+export default as("tableWrapper")(TableWrapper);
```

Comment:
It should render a `div`.

Reply:
noted. Is there a way to use `Base` and `component` together? Can I do something like;
`const TableWrapper = styled(Base)(Component)`

Reply:
Hmm. It'll not work. You should do this:
```jsx
const TableWrapper = styled(Base)`
  ...
`;

export default as("div")(TableWrapper);
```

Reply:
aah...sorry, I should have had a look at the code base to begin with. Let me know how it looks now.